charts/prometheus-postgres-exporter/README.md

```diff
@@ -30,21 +32,29 @@ _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documen
 ## Uninstall Chart
 
 ```console
-# Helm 3
+# Helm
 $ helm uninstall [RELEASE_NAME]
-
-# Helm 2
-# helm delete --purge [RELEASE_NAME]
 ```
 
 This removes all the Kubernetes components associated with the chart and deletes the release.
 
 _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
 
-## Upgrading Chart
+## Upgrading
+
+### To 2.0.0
```

Comment:
I would be very explicit here that the primary change in `2.0.0` is the Chart API `v1` to `v2`, which now requires Helm 3.